subsystem/subsystem_suite_test.go

```diff
@@ -53,16 +56,35 @@ func init() {
 		log.Fatal(err.Error())
 	}
 	userClientCfg := clientcfg(auth.UserAuthHeaderWriter("bearer " + Options.TestToken))
-	AgentClientCfg := clientcfg(auth.AgentAuthHeaderWriter("fake_pull_secret"))
+	adminUserClientCfg := clientcfg(auth.UserAuthHeaderWriter("bearer " + Options.TestTokenAdmin))
+	unallowedUserClientCfg := clientcfg(auth.UserAuthHeaderWriter("bearer " + Options.TestTokenUnallowed))
+	agentClientCfg := clientcfg(auth.AgentAuthHeaderWriter(FakePullSecret))
+	badAgentClientCfg := clientcfg(auth.AgentAuthHeaderWriter(WrongPullSecret))
 	userBMClient = client.New(userClientCfg)
-	agentBMClient = client.New(AgentClientCfg)
+	adminUserBMClient = client.New(adminUserClientCfg)
+	unallowedUserBMClient = client.New(unallowedUserClientCfg)
+	agentBMClient = client.New(agentClientCfg)
+	badAgentBMClient = client.New(badAgentClientCfg)
 
 	db, err = gorm.Open("postgres",
 		fmt.Sprintf("host=%s port=%s user=admin dbname=installer password=admin sslmode=disable",
 			Options.DBHost, Options.DBPort))
 	if err != nil {
 		logrus.Fatal("Fail to connect to DB, ", err)
 	}
+
+	var wiremock = &WireMock{
+		OCMHost:   Options.OCMHost,
+		TestToken: Options.TestToken,
+	}
+	err = wiremock.DeleteAllWiremockStubs()
```

Comment:
Seems strange that the stubs are installed as part of the clientcfg function.  It should be in context of the tests you wish to perform.  Shouldn't it be part of BeforeEach or similar?